index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
> Can you elaborate on what you mean with this? skip accepts a promise as well, so in that sense it is dynamic. But maybe you're hinting at something else :).

You said:

> So if you're in the task function, it's not possible to skip it afterwards because something failed.

I'd like that to be possible.